epictrack-api/src/api/schemas/request/work_request.py

```diff
@@ -192,7 +192,6 @@ class WorkStatusParameterSchema(RequestBodyParameterSchema):
 
     description = fields.Str(
         metadata={"description": "description of status"},
-        validate=validate.Length(max=500),
         required=True,
```

Comment:
Keep the limitation there. We don't want to remove it. We are waiting to hear from the ops team their requirements